client/my-sites/customer-home/locations/primary/index.jsx

```diff
@@ -11,25 +13,41 @@ import Webinars from 'my-sites/customer-home/cards/tasks/webinars';
 import FindDomain from 'my-sites/customer-home/cards/tasks/find-domain';
 import SiteSetupList from 'my-sites/customer-home/cards/tasks/site-setup-list';
 import { PerformanceTrackerStop } from 'lib/performance-tracking';
+import GoMobile from 'my-sites/customer-home/cards/tasks/go-mobile';
 
 const cardComponents = {
 	'home-task-site-setup-checklist': SiteSetupList,
 	'home-task-connect-accounts': ConnectAccounts,
 	'home-task-find-domain': FindDomain,
 	'home-task-webinars': Webinars,
+	'home-task-go-mobile-android': GoMobile,
+	'home-task-go-mobile-ios': GoMobile,
 };
 
 const Primary = ( { checklistMode, cards } ) => {
 	if ( ! cards || ! cards.length ) {
 		return null;
 	}
 
+	// Hard-coded. To be removed after D43129-code is merged
```

Comment:
Let's remove this before merging. It's safe to land this without D43129-code being merged (I'm planning to land it anyway just after we merge this PR).